OAuth/ResourceOwner/FiwareResourceOwner.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\OAuth\ResourceOwner;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+use Buzz\Message\RequestInterface as HttpRequestInterface;
+
+/**
+ * FiwareResourceOwner
```

Comment:
Why? That's done in every other ResourceOwner the same way! That's the way PhpStorm generates the doc comment.